src/lib/annotations/Annotator.js

```diff
@@ -541,25 +559,67 @@ class Annotator extends EventEmitter {
     }
 
     /**
-     * Binds event listeners for point annotation mode.
+     * Binds event listeners for annotation modes.
      *
      * @protected
+     * @param {string} mode - Current annotation mode
      * @return {void}
      */
-    bindPointModeListeners() {
-        const pointFunc = this.pointClickHandler.bind(this.annotatedElement);
-        const handlers = [
-            {
-                type: 'mousedown',
-                func: pointFunc,
-                eventObj: this.annotatedElement
-            },
-            {
-                type: 'touchstart',
-                func: pointFunc,
-                eventObj: this.annotatedElement
+    bindModeListeners(mode) {
+        let handlers = [];
```

Comment:
for each handler in handlers, we need to push the object to this.annotationModeHandler. (line 624) That way it will be unbound when unbindModeHandlers is called. Handlers is a local array.